core/common/src/main/java/alluxio/worker/block/BlockStore.java

```diff
@@ -111,7 +111,8 @@ BlockReader createBlockReader(long sessionId, long blockId, long offset,
    * @throws IOException if it fails to get block reader
    */
   BlockReader createUfsBlockReader(long sessionId, long blockId, long offset, boolean positionShort,
-                                   Protocol.OpenUfsBlockOptions options) throws IOException;
+                                   Protocol.OpenUfsBlockOptions options)
+      throws IOException;
```

Comment:
The indentation is still not fully correct
```suggestion
      Protocol.OpenUfsBlockOptions options) throws IOException;
```